README-cdk.md

```diff
@@ -262,10 +277,12 @@ Now use the `kubefed2` tool to register (*join*) the two clusters:
              --host-cluster-context cluster1 \
              --add-to-registry \
              --v=2
+~~~
 
 Verify that the federated clusters are registered and in a ready state (this
 can take a moment):
 
+~~~yaml
```

Comment:
This isn't YAML, it's a command and its output... formatting it as YAML might make it look nicer but also different from what people will see. In this case I would tend to prefer leaving the output as is...

I was actually wondering if we should emphasize the `ClusterReady` part of the output, but still not sure if we should format everything as if it were YAML

Reply:
ok, we can revert it as using 'sh'
```suggestion
~~~sh
```